### PR TITLE
Add documentation to note there are extension changes for oopbas

### PR DESCRIPTION
**Pull request by robmen**
Also add a note that building bundles using wix.exe requires updating to the new `WixToolset.BootstrapperApplications.wixext`.

Fixes wixtoolset/issues#8155

### CODE REVIEW

**Comment by robmen** [comment]
Hmm. The auto-merge didn't wait for approval by an assigned reviewer. That's not right.

File: src/Docusaurus/docs/fivefour/oopbas.md
Comment (barnson) [inline src/Docusaurus/docs/fivefour/oopbas.md:83]: > `PackageReferences` in MSBuild.
`PackageReferences` in MSBuild projects.

> Note: this means that if using `wix.exe` to add extensions you will need to use the new name `WixToolset.BootstrapperApplications.wixext`.
Note: When using `wix.exe`, you need to use the new name `WixToolset.BootstrapperApplications.wixext`.

File: src/Docusaurus/docs/fivefour/index.md
Comment (barnson) [inline src/Docusaurus/docs/fivefour/index.md:203]: > applications](/docs/fivefour/oopbas/)
applications](oopbas.md)

**Review by barnson — CHANGES_REQUESTED** [review]
Otherwise lgtm.

**Pull request merged**
